content/responses/file.yml

```diff
@@ -146,8 +146,8 @@ allOf:
           the `fields` query parameter to get the metadata by ID.
 
           The ID for a metadata template will be in the format
-          `metadata.<scope>.<templatename>`, for example
-          `metadata.global.properties`.
+          `metadata.<scope>.<templatename>.$id`, for example
```

Comment:
Not sure we want/need to add this $id here.

Reply:
I thought it would be a better example because specifying metadata.<scope>.<templatename> isn't actually a single field.

Reply:
Yes, but in the existing APIs that's how this works atm. This response resource is shared across a few APIs. We might need to do something clever to explain how metadata works in MDQ vs other APIs